orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -708,6 +707,7 @@ TEST(GradientCheckerTest, SplitGrad) {
 template <typename T>
 static std::vector<std::vector<T>> GetRandomValuesForMaxPool(const std::vector<TensorInfo>& infos) {
   std::vector<std::vector<T>> datas(infos.size());
+  std::random_device rd;
```

Comment:
>std::random_device rd; [](http://example.com/codeflow?start=2&length=22)

can we use generator initialized with GetTestRandomSeed()? then it's easier to reproduce failures

Reply:
Done.